src/minecraftctl.sh

```diff
@@ -17,6 +17,8 @@ RCON_CMD="mcrcon -P ${RCON_PORT}"
 if [ -n "$RCON_PASSWORD" ]
 then
     RCON_CMD="${RCON_CMD} -p ${RCON_PASSWORD}"
+else
+    RCON_CMD="${RCON_CMD} -p minecraft"
```

Comment:
Why was this bit added? If the password was not set I would expect that we do not specify any password in the rcon cmd.

Reply:
Because if there isn't a password specified, the docker container uses the default password, while your code was assuming no password. So, when there wasn't a password specified, the script was unable to stop the server due to authentication failure with mcrcon. That bit was just added to set the password to the default if not otherwise set.